### PR TITLE
Add edge cases for DeletePayment integration testing

### PR DESCRIPTION
**Pull request by enamourous**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeletePaymentCommandIntegrationTest.java [hunk 1]
```diff
@@ -92,4 +93,28 @@ public void execute_invalidPaymentIndex_throwsCommandException() {
...
+        Index personIndex = Index.fromOneBased(1);
+        Index paymentIndex = Index.fromOneBased(1);
+
+        DeletePaymentCommand command = new DeletePaymentCommand(List.of(personIndex), paymentIndex);
+
+        assertCommandFailure(command, emptyModel, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_invalidPersonIndex_throwsCommandException() {
```
Comment (dnt0n) [inline src/test/java/seedu/address/logic/commands/DeletePaymentCommandIntegrationTest.java]: Good test coverage for edge cases
